kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -869,7 +867,7 @@ def kube_limitrange(self, metric, scraper_config):
         else:
             self.log.error("Metric type %s unsupported for metric %s", metric.type, metric.name)
 
-    def count_objects_by_tags(self, metric, scraper_config):
+    def count_values_by_tags(self, metric, scraper_config):
         """ Count objects by allowed tags and submit counts as gauges. """
```

Comment:
Nit: could rephrase the comment to mention the difference between count_objects and count_values

Reply:
renamed to `sum_values_by_tags` to be clearer